diesel/src/connection/statement_cache.rs

```diff
@@ -1,3 +1,95 @@
+//! A primer on prepared statement caching in Diesel
+//! ------------------------------------------------
```

Comment:
If this were every actually rendered, it'd be the third h1 on the page and my 8 years out of date SEO skills say that there should only be one h1 per page